content/cadre.hbs

```diff
@@ -84,7 +73,7 @@
 							<figure class="demayo"></figure>
 						</div>
 						<div class="card-content">
-							<h1>Technical Sergeant Demayo</h1>
+							<h1>CTechnical Sergeant Demayo</h1>

```

Comment:
Looks like a typo here. Otherwise looks good

Reply:
Fixed it, thank you!